opentreemap/treemap/udf.py

```diff
@@ -630,10 +674,7 @@ def _validate_single_datatype(self, datatype):
             raise ValidationError(_('invalid datatype'))
 
         if datatype['type'] in ('choice', 'multichoice'):
-            choices = datatype.get('choices', None)
-
-            if choices is None:
-                raise ValidationError(_('Missing choices key'))
```

Comment:
Good catch. I'm going to add some test coverage around this and update the WM model so that the UDF definition contains an empty choices field.